SOLR-16699: Expose creationTimeMillis in COLSTATUS API (#2226)
apache/solr: solr/CHANGES.txt

* SOLR-17119: When registering or updating a ConfigurablePlugin through the `/cluster/plugin` API,
  config validation exceptions are now propagated to the callers. (Yohann Callea)

> * SOLR-16699: Add Collection creation time to the Collection Status API response (Julien Pilourdault, Paul McArthur, David Smiley)

Reviewer: ```suggestion
* SOLR-16699: Add Collection creation time to CLUSTERSTATUS and COLSTATUS API responses
   (Julien Pilourdault, Paul McArthur, David Smiley)
```
